onyo/lib/commands.py

```diff
@@ -471,57 +450,47 @@ def onyo_get(inventory: Inventory,
              match: list[Callable[[dict], bool]] | None = None,
              keys: list[str] | None = None,
              sort: dict[str, sort_t] | None = None) -> list[dict]:
-    r"""Query the repository for information about assets.
+    r"""Query the key-values of assets.
 
     Parameters
     ----------
     inventory
-      The inventory to query.
+        The Inventory to query.
     include
-      Limits the query to assets underneath these paths.
-      Paths can be assets and directories.
-      If no paths are specified, the inventory root is used as default.
+        Assets or directories to query.
+        Default: inventory root
     exclude
-      Paths to exclude, meaning that assets underneath any of these are not
-      being returned. Defaults to `None`. Note, that `depth` only applies to
-      `include`, not to `exclude`. `depth` and `exclude` are different ways
-      of limiting the results.
+        Assets or directories to exclude from the query.
     depth
-      Number of levels to descend into. Must be greater or equal 0.
-      If 0, descend recursively without limit.
+        Descend up to **depth** levels into the directories specified by
+        ``include``. A depth of ``0`` descends recursively without limit.
     machine_readable
-      Whether to print the matching assets as TAB-separated lines,
-      where the columns correspond to the `keys`. If `False`,
-      print a table meant for human consumption.
+        Toggle whether to print the output machine-friendly (no headers and
+        separate values with a single tab) vs more human-friendly output
+        (headers and padded whitespace to align columns).
     match
-      Callables suited for use with builtin `filter`. They are
-      passed an asset dictionary and expected to return a `bool`,
-      where `True` indicates a match. The result of the query
-      consists of all assets that are matched by all callables in
-      this list. One can match keys that are not in the output.
+        Callables suited for use with builtin :py:func:`filter`. They are passed
+        an asset dictionary and expected to return a ``bool``.
```

Comment:
"They are passed an asset dictionary" sounds incorrect. Maybe it should have been "passed as an"?

Reply:
In this case, the wording is correct: an Asset Dictionary is passed to the Callable.

Reply:
Ohh, this means "They get an asset dictionary passed"? Somehow I needed to re-read it 5 more times to get it. The active phrasing of "are passed", even though "They" (the callable) is receiving them passive did through me off.